guides/common/modules/con_kerberos-configuration-in-web-browsers.adoc

```diff
@@ -22,7 +22,7 @@ ad_gpo_map_service = +foreman
 ----
 
 Here, _foreman_ is the PAM service name.
-ifndef::orcharhino[]
-For more information on GPOs, please refer to the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/windows_integration_guide/sssd-gpo[{RHEL} Windows Integration Guide].
+ifdef::satellite[]
+For more information on GPOs, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#applying-group-policy-object-access-control-in-rhel_managing-direct-connections-to-ad[{RHEL} Integrating RHEL systems directly with Windows Active Directory guide].
```

Comment:
```suggestion
For more information on GPOs, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#applying-group-policy-object-access-control-in-rhel_managing-direct-connections-to-ad[How SSSD interprets GPO access control rules] in _Integrating RHEL systems directly with Windows Active Directory_.
```
The link currently leads to section named like this.

Reply:
Disregarding other comments I made, this link still needs to be updated because it the name of the section is currently incorrect.